### ISSUE TITLE
Problems with resume trainning

### ISSUE THREAD

**Issue by bdlneto** [ptran1203/pytorch-animeGAN#5]
Hi!

I'm facing a issue when i have to resume the trainning, even when I reach 80+ epochcs, the transformer has no effect on the images with the trained weights. I tried Hayao and other custom datasets, here are images with 15 epochs (without resume), 30, 40, 60 and  80 epochs with Hayao Dataset, and the changes are barely visible, if existent, I'm using a copy of the Google Collab Notebook.

![1 (1)_anime_15_epochs](https://user-images.githubusercontent.com/3859084/142866565-2c5bc475-8600-4aca-bb58-c84a577cf899.jpg)
![1 (1)_anime_30_epochs](https://user-images.githubusercontent.com/3859084/142866576-93cafd2f-2467-4e20-a257-5839d969a651.jpg)
![1 (1)_anime_40_epochs](https://user-images.githubusercontent.com/3859084/142866579-07d3553f-1b27-47a9-a855-4f922a03bde7.jpg)
![1 (1)_anime_60_epochs](https://user-images.githubusercontent.com/3859084/142866580-c6208915-cb36-45a2-b2ce-6125c7147b4a.jpg)
![1 (1)_anime_80_epochs](https://user-images.githubusercontent.com/3859084/142866585-a5d20576-b34d-4697-b058-ac940aa9a67b.jpg)

Is there anything I'm doing wrong with the process? Here are the params used for trainning:

!python3 train.py --dataset 'Hayao'\
                  --batch 6\
                  --debug-samples 0\
                  --init-epochs 10\
                  --epochs 100 (Changed at each trainning resume)\
                  --checkpoint-dir {ckp_dir}\
                  --save-image-dir {save_img_dir}\
                  --save-interval 1\
                  --gan-loss lsgan\
                  --init-lr 0.0001\
                  --lr-g 0.00002\
                  --lr-d 0.00004\
                  --wadvd 10.0\
                  --wadvg 10.0\
                  --wcon 1.5\
                  --wgra 3.0\
                  --wcol 70.0\
                  --resume GD\
                  --use_sn\
